### PR TITLE
Feature/446 gridlines

### LINKED ISSUE #446: [iOS] Timetable Grid: Grid lines
Grid lines are missing from the grid view. These should be added in as per figma spec:

Current | Should Be
:--: | :--:
スクリーンショット 2024-08-14 23 22 33 |  スクリーンショット 2024-08-14 23 34 07

Figma Ref:
https://www.figma.com/design/XUk8WMbKCeIdWD5cz9P9JC/DroidKaigi-2024-App-UI?node-id=55495-40872&t=byaoWt5YjzQK8OHa-0

### PR DESCRIPTION
**Pull request by charles-b-stb**
## Issue
- close #446

## Overview (Required)
- Simply adding dotted gridlines to the grid style timetable view. 

Draft: The vertical lines are not yet working and I am looking for advice.
Branched off of the [Grid UI pull request](https://github.com/DroidKaigi/conference-app-2024/pull/367) so that will need to be merged first.

## Links
- 

## Screenshot (Optional if screenshot test is present or unrelated to UI)
Before | After
:--: | :--:
 | 

## Movie (Optional)
Before | After
:--: | :--:
<video src="" width="300" > | <video src="" width="300" >

### CODE REVIEW

**Comment by charles-b-stb** [comment]
Let's try just adding horizontal graph lines for now. 

If anyone can think of a reasonable way to add vertical lines I would appreciate it. 🙇

File: app-ios/Sources/TimetableFeature/TimetableListView.swift [hunk 1]
```diff
@@ -192,6 +194,39 @@ struct TimeGroupMiniList: View {
...
+    var body: some View {
+        let shape = LineShape(axis: axis)
+            .stroke(style: .init(dash: [2]))
+            .foregroundStyle(AssetColors.Outline.outlineVariant.swiftUIColor)
+        if axis == .horizontal {
+            shape.frame(height: 1)
+        } else {
+            shape.frame(width: 1).padding(0)
+        }
+        //shape
```
Comment (MrSmart00) [inline app-ios/Sources/TimetableFeature/TimetableListView.swift:209]: What's this comment🤔?

File: app-ios/Sources/TimetableFeature/TimetableListView.swift [hunk 2]
```diff
@@ -192,6 +194,39 @@ struct TimeGroupMiniList: View {
...
+    }
+}
+
+struct LineShape: Shape {
+    public var axis: Axis
```
Comment (MrSmart00) [inline app-ios/Sources/TimetableFeature/TimetableListView.swift:214]: this property is 'let' more than better👍

File: app-ios/Sources/TimetableFeature/TimetableListView.swift [hunk 3]
```diff
@@ -192,6 +194,39 @@ struct TimeGroupMiniList: View {
     }
 }
 
+struct DashedDivider: View {
+    public var axis: Axis
```
Comment (MrSmart00) [inline app-ios/Sources/TimetableFeature/TimetableListView.swift:198]: this property is 'let' more than better👍

**Review by MrSmart00 — COMMENTED** [review]
THX!!
I left comments!
please check them🙏

File: app-ios/Sources/TimetableFeature/TimetableListView.swift (hunk 1 above)
Comment (charles-b-stb) [reply app-ios/Sources/TimetableFeature/TimetableListView.swift:209]: Removed

**Comment by charles-b-stb** [comment]
@MrSmart00 Responded to comments, thanks!

File: app-ios/Sources/TimetableFeature/TimetableListView.swift (hunk 3 above)
Comment (MrSmart00) [reply app-ios/Sources/TimetableFeature/TimetableListView.swift:197]: you should change here too🙏

**Comment by charles-b-stb** [comment]
@MrSmart00 Sorry I missed that last time (it looked too similar to the other item I corrected and I did not notice they were separate the first time I looked at it.)

Fixed now.

**Review by MrSmart00 — APPROVED** [review]
Thanks your commits!
LGTM👍

**Pull request merged**